Microsoft.Identity.Web/Resource/ScopesRequiredByWebAPIExtension.cs

```diff
@@ -13,12 +13,17 @@ namespace Microsoft.Identity.Web.Resource
     public static class ScopesRequiredByWebAPIExtension
     {
         /// <summary>
-        /// When applied to a Controller, verifies that the user authenticated in the Web API has any of the
-        /// accepted scopes
+        /// When applied to an <see cref="HttpContext"/>, verifies that the user authenticated in the Web API has any of the
+        /// accepted scopes. If the authentication user does not have any of these <paramref name="acceptedScopes"/>, the
+        /// method throws an HTTP Unauthorized with the message telling which scope are missing
         /// </summary>
         /// <param name="acceptedScopes">Scopes accepted by this API</param>
         public static void VerifyUserHasAnyAcceptedScope(this HttpContext context, IEnumerable<string> acceptedScopes)
         {
+            if (acceptedScopes == null)
+            {
+                throw new ArgumentNullException("acceptedScopes");
```

Comment:
Nit: prefer nameof(acceptedScopes) - this way if you rename the param, the exception message will also be updated.

Reply:
Done. Thanks!